### PR TITLE
feat(DOJO): Create main mentor page

### PR DESCRIPTION
**Pull request by flacial**
Related to #2029 

### Changes

- Create the main mentor page under `curriculum/[lessonSlug]/mentor` which is a *literal* modified copy of the [exercises page](https://c0d3.com/exercises/js1) including its code and test file.
- Add another path to the `NavBar` in the exercises page that link to `/curriculum/[lessonSlug]/mentor`
- Move the page for adding an exercise to `/mentor/addExercise`

<details>
<summary>
Page Image
</summary>
image

</details>

### Subsequent changes

1. Create a query to get all the mentor's created exercises based on the user ID
2. Modify the exercise card to not show ANSWERED or NOT ANSWERED states
3. Add a back button to the `addExercise` page

### Testing

1. Go to `/curriculum/js0/mentor`
2. Confirm the pages show up correctly
3. Click `MENTOR EXERCISES` in the `NavBar`
4. Confirm the `addExercise` page show up correctly

### CODE REVIEW

File: pages/curriculum/[lessonSlug]/mentor/addExercise/index.tsx [hunk 1]
```diff
@@ -0,0 +1,221 @@
+import React, { useMemo, useState } from 'react'
```
Comment (flacial) [inline pages/curriculum/[lessonSlug]/mentor/addExercise/index.tsx:1]: Nothing changed. Just moved the file.

File: __tests__/pages/curriculum/[lessonSlug]/mentor/addExercise/index.test.js [hunk 2]
```diff
@@ -0,0 +1,306 @@
+jest.mock('@sentry/nextjs')
```
Comment (flacial) [inline __tests__/pages/curriculum/[lessonSlug]/mentor/addExercise/index.test.js:1]: Nothing new. Just moved the file.

File: pages/curriculum/[lessonSlug]/mentor/index.tsx [hunk 3]
```diff
@@ -1,221 +1,132 @@
...
+            ADD EXERCISE
+          </NewButton>
+        </div>
+      </div>
+      <div className={styles.exerciseList__container}>
+        {exercises.map((exercise, i) => (
+          <ExercisePreviewCard
+            key={i}
+            moduleName={exercise.moduleName}
+            state={'ANSWERED'}
```
Comment (flacial) [inline pages/curriculum/[lessonSlug]/mentor/index.tsx:116]: Temporarily set to `ANSWERED`. Will be fixed in a different PR.

File: pages/exercises/[lessonSlug].tsx [hunk 4]
```diff
@@ -39,7 +39,11 @@ const Exercises: React.FC<QueryDataProps<GetExercisesQuery>> = ({
       ? [{ text: 'lessons', url: currentLesson.docUrl }]
       : []),
     { text: 'challenges', url: `/curriculum/${currentLesson.slug}` },
-    { text: 'exercises', url: `/exercises/${currentLesson.slug}` }
+    { text: 'exercises', url: `/exercises/${currentLesson.slug}` },
+    {
+      text: 'mentor exercises',
+      url: `/curriculum/${currentLesson.slug}/mentor/`
```
Comment (flacial) [inline pages/exercises/[lessonSlug].tsx:45]: We should use a single `NavBar` across the curriculum pages. It'll be done in a different PR.

**Review by bryanjenningz — APPROVED** [review]

**Pull request merged**